--- farstream-0.2.4/farstream/fs-conference.c.orig	2013-03-26 18:12:17.000000000 +0100
+++ farstream-0.2.4/farstream/fs-conference.c	2014-06-15 10:02:31.553668384 +0200
@@ -43,7 +43,7 @@
  *
  * This will communicate asynchronous events to the user through #GstMessage
  * of type #GST_MESSAGE_ELEMENT sent over the #GstBus.
- * </para>
+ *
  * <refsect2><title>The "<literal>farstream-error</literal>" message</title>
  * |[
  * "src-object"       #GObject           The object (#FsConference, #FsSession or #FsStream) that emitted the error
@@ -54,7 +54,6 @@
  * The message is sent on asynchronous errors.
  * </para>
  * </refsect2>
- * <para>
  */
 
 
--- farstream-0.2.4/farstream/fs-session.c.orig	2014-05-05 20:55:48.000000000 +0200
+++ farstream-0.2.4/farstream/fs-session.c	2014-06-15 10:12:03.266989800 +0200
@@ -39,7 +39,7 @@
  *
  * This will communicate asynchronous events to the user through #GstMessage
  * of type #GST_MESSAGE_ELEMENT sent over the #GstBus.
- * </para>
+ *
  * <refsect2><title>The "<literal>farstream-send-codec-changed</literal>"
  *   message</title>
  * <table>
@@ -132,7 +132,7 @@
  * telephony event has stopped.
  * </para>
  * </refsect2>
- * <para>
+ *
  */
 
 #ifdef HAVE_CONFIG_H
--- farstream-0.2.4/farstream/fs-stream.c.orig	2014-05-05 20:55:48.000000000 +0200
+++ farstream-0.2.4/farstream/fs-stream.c	2014-06-15 10:12:42.880322308 +0200
@@ -36,7 +36,7 @@
  *
  * This will communicate asynchronous events to the user through #GstMessage
  * of type #GST_MESSAGE_ELEMENT sent over the #GstBus.
- * </para>
+ *
  * <refsect2><title>The "<literal>farstream-new-local-candidate</literal>" message</title>
  * |[
  * "stream"           #FsStream          The stream that emits the message
@@ -93,7 +93,6 @@
  * This message is emitted the state of a component of a stream changes.
  * </para>
  * </refsect2>
- * <para>
  */
 
 #ifdef HAVE_CONFIG_H
--- farstream-0.2.4/gst/fsrtpconference/fs-rtp-session.c.orig	2014-05-05 20:55:48.000000000 +0200
+++ farstream-0.2.4/gst/fsrtpconference/fs-rtp-session.c	2014-06-15 10:14:38.133653238 +0200
@@ -30,7 +30,7 @@
  * fs_conference_new_session() on a #FsRtpConference. It can be either
  * Audio or Video. It also represents data send with one and only one
  * SSRC (although if there is a SSRC collision, that SSRC may change).
- * </para>
+ *
  * <refsect2><title>Codec profiles</title>
  * <para>
  * It is possible to define "codec profiles", that is non-autodetected
@@ -50,7 +50,7 @@
  * Also, it is possible to declare profiles with only a decoding pipeline,
  * you will only be able to receive from this codec, the encoding may be a
  * secondary pad of some other codec.
- * </para></refsect2><para>
+ * </para></refsect2>
  */
 
 #ifdef HAVE_CONFIG_H
